presto-main/src/test/java/com/facebook/presto/sql/planner/TestLogicalPlanner.java

```diff
@@ -267,6 +293,113 @@ private Plan plan(String sql, LogicalPlanner.Stage stage)
         }
     }
 
+    private static void assertEqualPlans(Plan actual, Plan expected)
```

Comment:
There are already some utilities for making assertions over plan structures. Take a look at `com.facebook.presto.sql.planner.assertions.PlanAssert`.